Dockerfile

```diff
@@ -1,3 +1,9 @@
-FROM node:4-onbuild
+FROM node:4.2
 
-CMD ["node", "app.js"]
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+COPY . /usr/src/app
+RUN npm instal
```

Comment:
Wow, what worries me is not the typo but that the build did this:

```
Step 3 : COPY . /usr/src/app
 ---> ef13a521b477
Removing intermediate container 0a5bdf7fc996
Step 4 : RUN npm instal
 ---> Running in 527a6136fe1a
npm info it worked if it ends with ok
npm info using npm@2.14.7
npm info using node@v4.2.2
```

I don't know enough node, but to me that should have been a failure :)